### PR TITLE
Cleanup gitignore and make linter pass

### PR DESCRIPTION
**Pull request by nichochar**
Added a standard gitignore and made sure that the linter passes when you run the commands described in `CONTRIBUTING.md`

### CODE REVIEW

File: snappass/main.py [hunk 1]
```diff
@@ -23,8 +22,12 @@
 }
 
 
+def _id():
+    return uuid.uuid4().hex
+
+
 def set_password(password, ttl):
-    key = id_()
+    key = _id()
```
Comment (jparise) [inline snappass/main.py]: Is `_id()` used (or mocked) elsewhere? If not, it seems like we could just call `uuid.uuid4().hex` directly here.

**Comment by davedash** [comment]
Thanks for these.

I no longer have commit access to this repo, but I’ll probably take this and the PRs, etc that are piling up and move them somewhere under my control.

-d

-- 
Dave Dash


On July 18, 2016 at 11:54:47 AM, Nicholas Charriere (notifications@github.com) wrote:

Added a standard gitignore and made sure that the linter passes when you run the commands described in CONTRIBUTING.md

You can view, comment on, or merge this pull request online at:

  https://github.com/pinterest/snappass/pull/11

Commit Summary

Add virtualenv env/ to gitignore
Make flake8 test pass
File Changes

M .gitignore (50)
M snappass/main.py (13)
M tests.py (5)
Patch Links:

https://github.com/pinterest/snappass/pull/11.patch
https://github.com/pinterest/snappass/pull/11.diff

File: snappass/main.py (hunk 1 above)
Comment (nichochar) [inline snappass/main.py]: I agree with this, will update.

**Comment by jparise** [comment]
👍

**Pull request merged**